admin_scripts/ubuntu/ubuntu_16.04/os2display-specific/auto_activate_chrome.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+"""
+Script for activating an OS2display screen on Google Chrome. Arguments: [url, activation_code]
+"""
+
+__author__ = "Danni Als"
+__copyright__ = "Copyright 2019, Magenta Aps"
+__credits__ = ["Allan Grauenkjaer"]
+__license__ = "GPL"
+__version__ = "0.2.0"
+__maintainer__ = "Magenta"
+__email__ = "danni@magenta.dk"
+__status__ = "Production"
+
+import os
+import glob
+import sys
+import stat
+import subprocess
+from urlparse import urlparse
+
+subprocess.call([sys.executable, "-m", "pip", "install", 'wget'])
+subprocess.call([sys.executable, "-m", "pip", "install", 'selenium'])
+subprocess.call([sys.executable, "-m", "pip", "install", 'plyvel'])
+
+print('Installed wget and selenium.')
```

Comment:
I am sort of amazed that this works. :-)

(Some groups of scripts -- like those that handle Princh, for example, are broken up into two: one installs the dependencies and the other carries out the task. Which approach do we prefer -- separate scripts or integrated scripts?)

Reply:
I think scripts should be divided into seperate scripts when it makes sense. So fx Google Chrome must have an install script and afterwards you run different setup scripts. 

Installing wget selenium and plyvel is something only this script uses, so therefore I do not think it should be in its own script...